ping/ping_common.c

```diff
@@ -794,6 +795,9 @@ int gather_statistics(struct ping_rts *rts, uint8_t *icmph, int icmplen,
 		if (pr_reply)
 			pr_reply(icmph, cc);
 
+		if (rts->opt_verbose && rts->ident != -1)
+			printf(_(" ident=%d"), ntohs(rts->ident));
```

Comment:
I thought it'd be interesting to print the ident always on `-v`. But that would change the output formatting, thus better to keep it as it as you suggested (only when ident specified by `-e`).

Reply:
My thoughts exactly